bin/cdxgen.js

```diff
@@ -646,7 +646,7 @@ const checkPermissions = (filePath) => {
     }
   }
   // Automatically submit the bom data
-  if (options.serverUrl && options.serverUrl != true && options.apiKey) {
```

Comment:
This was catching the case where serverUrl was passed as a string "true". Can you try replicating by passing `--server-url` without any values?

Reply:
I'm not sure I'm following, sorry...

Reply:
!== true is a stricter check which won't work if the input is string "true". yargs is setting a string "true" for empty serverUrl values.

Reply:
@setchy can we retain `!=` in this line alone and commit the rest?

Reply:
Reverted this and added a biome ignore comment